platformio.ini

```diff
@@ -23,7 +23,7 @@
 [platformio]
 src_dir      = TFT
 boards_dir   = buildroot/boards
-default_envs = BIGTREE_TFT35_V3_0
+default_envs = BIGTREE_TFT35_V2_0
```

Comment:
This should be reverted back to `BIGTREE_TFT35_V3`

Reply:
ok i change it